grunt-configs/concat.js

```diff
@@ -1,10 +1,23 @@
 module.exports = function (grunt, options) {
     return {
+        options: {
+            process: function (src, path) {
+                // make sure omniture handles an error
+                // this image fallback is duplicated in /common/app/views/fragments/omnitureScript.scala.html
+                if (path.match(/vendor\/omniture\.js/)) {
+                    src = 'try {' + src + '} catch (e) {(new Image()).src = window.guardian.config.page.beaconUrl + "/count/omniture-library-error.gif";}';
```

Comment:
This is a little shim-like, i think we assess what we have left as a 'standalone analytics application', and build that as an asset that can be pre-pended to `app.js`